virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -586,3 +586,30 @@ def fill_null_in_vm(vm, target, size_value=500):
             raise exceptions.TestError("Error happened when executing command:\n%s" % cmd)
     except Exception as e:
         raise exceptions.TestError(str(e))
+
+
+def check_virtual_disk_io(vm, partition):
```

Comment:
How about set a default value for partition, such as vdb or sdb

Reply:
Frankly speaking, I don't see benefit to set partition as one default value.
Without setting default value for partition, it help remind users that partition parameter need be given in any situation.